gherkin-utils/java/gherkin-utils/src/main/java/gherkin/utils/walker/model/DefaultFilters.java

```diff
@@ -0,0 +1,25 @@
+package gherkin.utils.walker.model;
+
+import io.cucumber.messages.Messages;
+
+public abstract class DefaultFilters {
```

Comment:
This class should be an interface. E.g. `Filter` with an `RejectAllFilter`  and `AcceptAllFilter` implementation.

Reply:
My first implementation I also use interface like you suggest. But I think with abstract class, when we want to custom the ```Filter```, we don't need to implement all method of the interface ```Filter```. E.g:
```
GherkinDocumentWalker walker = new GherkinDocumentWalker(new RejectAllFilters() {
            @Override
            public boolean acceptScenario(Messages.GherkinDocument.Feature.Scenario scenario) {
                return scenario.getName().equals("Earth");
            }
        }, new DefaultHandlers() {
        });
```